securesign/secure-sign-operator
# feat: add service monitors to log server and log signer

This pr adds service monitors to log server and log signer.

# Testing

1. Deploy operator
2. Create CRD with this in it:
```
  trillian:
    monitoring:
      enabled: true
```
3. Targets will come up, and you can see metrics in Observe ->  Metrics
